### PR TITLE
Add Topaz Sharpen AI.app v4.1.0

### PR DESCRIPTION
**Pull request by LoreleiAurora**
After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online topaz-sharpen-ai` is error-free.
- [x] `brew style --fix topaz-sharpen-ai` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask topaz-sharpen-ai` worked successfully.
- [x] `brew install --cask topaz-sharpen-ai` worked successfully.
- [x] `brew uninstall --cask topaz-sharpen-ai` worked successfully.

### CODE REVIEW

**Comment by fblupi** [comment]
> Thank you for your contribution to `homebrew-cask` @fblupi.

That wasn't me! 😝

The PR has been made by @LoreleiAurora

**Comment by miccal** [comment]
> > Thank you for your contribution to `homebrew-cask` @fblupi.
> 
> That wasn't me! 😝
> 
> The PR has been made by @LoreleiAurora

Sorry @fblupi!

File: Casks/topaz-sharpen-ai.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "topaz-sharpen-ai" do
+  version "4.1.0"
+  sha256 :no_check
+
+  url "https://downloads.topazlabs.com/deploy/TopazSharpenAI/latest/TopazSharpenAI-Full-Installer.dmg"
+  name "Topaz Sharpen AI"
+  desc "AI-powered image sharpener that produces natural results"
```
Comment (Amorymeltzer) [inline Casks/topaz-sharpen-ai.rb:7]: This sounds like a marketing slogan, and indeed appears to be the tagline from the website, which is supposed to be avoided in `desc`.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**